velox/functions/lib/SubscriptUtil.h

```diff
@@ -23,6 +23,22 @@
 
 namespace facebook::velox::functions {
 
+namespace {
```

Comment:
No anonymous namespace in header.  Either put it in `detail` or the main class `SubscriptImpl`